parser.go

```diff
@@ -178,8 +178,13 @@ func scanNode(cx *parseContext, sc *pkg.Scanner, name string) (Node, error) {
 		case pkg.SEMICOLON:
 			done = true
 		case pkg.WS:
-			// FIXME: handle other types of newline control characters (see the spec)
-			if strings.Contains(lit, "\n") || strings.Contains(lit, "\r") {
+			if strings.Contains(lit, "\n") || // newline
+				strings.Contains(lit, "\r") || // carriage return
+				strings.Contains(lit, "\r\n") || // carriage return newline
+				strings.Contains(lit, "\f") || // form feed
+				strings.Contains(lit, "\u0085") || // next line
+				strings.Contains(lit, "\u2028") || // line separator
+				strings.Contains(lit, "\u2029") { // paragraph separator
```

Comment:
I think this must be handled as well in the `scanString` method below.
If you put all these possible "newline" characters into an array or something (global variable, perhaps)
it should be fairly straightforward.

Reply:
OK I updated `scanString` as well but the unit test I added is failing for line separator and paragraph separator in quoted string literal values with an EOF error. Not sure if that's a parsing bug that needs to be fixed or a bug in this change. Could you have a look?

Reply:
Awesome! I can take a look .

Reply:
I found the issue. Adding the following case to the switch in scanString should do the trick:
```go
		case pkg.RAWSTR_OPEN, pkg.RAWSTR_HASH_OPEN:
			buf.WriteString(lit[:len(lit)-1])
			done = true
```